tests/e2e_tests/utils.py

```diff
@@ -3,9 +3,10 @@
 import subprocess
 import sys
 
-import requests
-from substrateinterface import Keypair
 from typing import List
+
+from substrateinterface import Keypair
```

Comment:
FYI: we have `Keypair` imported into `bittensor/__init__.py`. You can use it as `bittensor.Keypair`.

Reply:
Got it. Importing from bittensor